advocacy_docs/partner_docs/CohesityDataProtectforPostgreSQL/04-ConfiguringCohesityDataProtectforPostgreSQL.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: 'Configuring Cohesity DataProtect for PostgreSQL'
+description: 'Walkthrough on configuring Cohesity DataProtect for PostgreSQL'
+---
+
+Implementing Cohesity DataProtect for PostgreSQL with EDB Postgres Advanced Server requires the following components:
+
+- EDB Postgres Advanced Server 
+- Cohesity Backup Appliance
+- Cohesity Linux Agent 
+- Cohesity PostgreSQL Agent 
+
+### Prerequisites
+
+- A running EDB Postgres Advanced Server instance
+- Cohesity Backup Appliance installed and running on VMware
+- Cohesity agents installed on the same machine as your EDB Postgres Advanced Server database
+- The user account of the PostgreSQL instance must have database admin privileges
+- Login information for the Cohesity Web interface
+- Credentials that are used to access the EDB Postgres Advanced Server database
+- Java 1.8 or higher version is installed on the PostgreSQL nodes
+
+### Configure Cohesity DataProtect for PostgreSQL for EDB Postgres Advanced Server
+
+**Register and Manage the EDB Postgres Advanced Server Source:**
+
+1. Log on to the Cohesity console at the IP address that was used to create your VMware Cohesity Backup Appliance with the format: `https://ipaddress/login`
+
+   ![Cohesity Login Page](Images/CohesityLoginPage.png)
+
+2. When the Cohesity Dashboard login page appears, login with your Cohesity credentials.
+
+   ![Cohesity Login Credentials](Images/CohesityLoginCredentials.png)
+
+3. Once logged in, you will then be presented with the Cohesity Dashboard.
+
+   ![Cohesity Dashboard](Images/CohesityDashboard.png)
+
+4. Register your EDB Postgres Advanced Server database. From the main Dashboard screen select `Data Protection` and then select `Sources` on the left-hand side.
```

Comment:
This section is Registering your EPAS.
Can delete "Register your EDB Postgres Advanced Server database."